tests/test_decoders.py

```diff
@@ -79,6 +86,23 @@ def compress(body):
     assert response.read() == body
 
 
+@pytest.mark.parametrize("header_value", (b"deflate", b"gzip", b"br", b"identity"))
+def test_empty_content(header_value):
+    headers = [(b"Content-Encoding", header_value)]
+    response = httpx.Response(200, headers=headers, content=b"")
+    assert response.content == b""
+
+
+@pytest.mark.parametrize(
+    "decoder",
+    (BrotliDecoder, DeflateDecoder, GZipDecoder, IdentityDecoder, MultiDecoder),
```

Comment:
Don't really need to include `MutliDecoder` here - that's a different type of case.